sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_first_seen_v1/init.sql

```diff
@@ -1,144 +1,24 @@
-CREATE OR REPLACE TABLE
-  fxa_users_services_first_seen_v1
+CREATE TABLE IF NOT EXISTS
+  `moz-fx-data-shared-prod.firefox_accounts_derived.fxa_users_services_first_seen_v1`
 PARTITION BY
-  DATE(first_service_timestamp)
+  first_seen_date
 CLUSTER BY
   service,
-  user_id
+  os_name,
+  country
```

Comment:
I think clustering by user_id vs country would be more beneficial

Reply:
on the point @relud made, because we will have so few entries per user_id clustering on that field makes no sense here.